WebRoot/messages/ZmMsg.properties

```diff
@@ -3522,7 +3522,7 @@ toLabel = To:
 top = Top
 trash = Trash
 trustedAddrs = Trusted Addresses
-trustedAddrsDomains = Trusted Addresses / Domains
+trustedAddrsDomains = Trusted Addresses
```

Comment:
I think this will reduce information that is given to users, it will be hard for users to know that they can add domains also here